etna/lib/etna/clients/magma/client.rb

```diff
@@ -7,10 +7,12 @@
 module Etna
   module Clients
     class Magma
+      attr_reader :host
       def initialize(host:, token:, persistent: true)
         raise 'Magma client configuration is missing host.' unless host
         raise 'Magma client configuration is missing token.' unless token
         @etna_client = ::Etna::Client.new(host, token, routes_available: false, persistent: persistent)
+        @host = host
```

Comment:
Saving and exposing the host value on Magma client so that the create-project workflow can instantiate a new Magma client with an updated token that contains the new project's permissions.